css-color-5/Overview.bs

```diff
@@ -745,20 +745,20 @@ Unless the type of hue interpolation is ''specified'', both angles need to be co
 One way to do this is <code><i>θ</i> = ((<i>θ</i> % 360) + 360) % 360</code>.
 
 : ''shorter''
-:: Angles are adjusted so that θ₂ - θ₁ ∈ [0, 180). In pseudo-Javascript:
+:: Angles are adjusted so that θ₂ - θ₁ ∈ [-180, 180). In pseudo-Javascript:
 	<pre>
 	if (θ₂ - θ₁ >= 180) {
 		θ₁ += 360;
 	}
-	else if (θ₂ - θ₁ <= -180) {
+	else if (θ₂ - θ₁ < -180) {
 		θ₂ += 360;
 	}
 	</pre>
 
 : ''longer''
-:: Angles are adjusted so that θ₂ - θ₁ ∈ [180, 360). In pseudo-Javascript:
+:: Angles are adjusted so that θ₂ - θ₁ ∈ {(-360, -180], 0, (180, 360)}. In pseudo-Javascript:
 	<pre>
-	if (0 < θ₂ - θ₁ < 180) {
+	if (0 < θ₂ - θ₁ <= 180) {
```

Comment:
Why is this needed? Currently on the equal case, you just don't add 360. It's still well-defined. Is there a reason why we want to enter this clause?

Reply:
The reason for wanting to do this is to follow the principle from @svgeesus that angles that differ by 180 should go in the `decreasing` direction.  Without this change, the direction they go would depend on what angles they were, since it would be determined by the constraining to `[0, 360)`.

Reply:
After thinking about it some more, I think if θ₂ - θ₁ == 180 (so θ₂ == θ₁ + 180) then interpolation should be the same as `increasing`, whereas when θ₂ - θ₁ == -180 (so θ₂ == θ₁ - 180), interpolation should be the same as decreasing. This seems to be less surprise-inducing than going in the same direction regardless of whether it's θ₂ < θ₁ or θ₂ > θ₁.
I'll think of how to put that in these algorithms tomorrow (or if you want to give it a shot, fine by me!)

Reply:
Yes, @LeaVerou convinced me that basing the direction at exactly 180 on the sign of the angle difference made more sense.